src/groovy/au/org/emii/portal/proxying/ProxiedRequest.groovy

```diff
@@ -14,9 +14,9 @@ class ProxiedRequest extends ExternalRequest {
     def response
     def params
 
-    ProxiedRequest(request, response, params, proxyRedirectService) {
+    ProxiedRequest(request, response, params, proxyRedirectService, grailsApplication) {
```

Comment:
Why require the whole config? Wouldn't it be simpler to just receive the timeout value by itself?